src/rpc/RpcHandleProxy.ts

```diff
@@ -35,12 +35,32 @@ class RpcMethodsProxy {
                         return Reflect.get(target, prop, receiver);
                     }
                     if (typeof prop === "symbol") return;
-                    return (...args: any) => {
+                    return async (...args: any) => {
```

Comment:
this makes the usage ugly - it's really nice to be able to chain e.g. (`remote.service.method(args).broadcast()`)

Better to make the end functions async if you need to sign -> in `RpcHandler` make `broadcast, sendOne, sendMultiple` async and sign there - they all seralize the RPC and those functions are so small and don't change and then we can keep the nice `chaining` and awaiting them is not relevant - they will eventually just send out the message to the network which will regardless travel with some network delay which is aysnc anyway